src/yubihsm.rs

```diff
@@ -49,7 +46,7 @@ pub fn get_config() -> YubihsmConfig {
 #[cfg(not(feature = "yubihsm-mock"))]
 pub fn create_hsm_connector() -> Box<dyn Connector> {
     // TODO: `HttpConnector` support
-    let connector = UsbConnector::new(&get_config().usb_config()).unwrap_or_else(|e| {
+    let connector = UsbConnector::create(&get_config().usb_config()).unwrap_or_else(|e| {
```

Comment:
These names changed upstream after clippy started to warn for `new` that doesn't return `Self`